Initialtask/src/Service/FileParserService.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Service;
+
+use App\Repository\FileParserRepository;
+
+class FileParserService
```

Comment:
Guess, that it doesn't do any significant job
Do we actually need it?

Reply:
Yes, we need.
	Service used in ProductsImportCommand.php
	at 	line 53  	$fileData = $this->fileParserService->loadData($filePath);